Probesys/bileto
# doc: Add documentation to deploy in production

Changes proposed in this pull request:

- Fix the name of a migration
- Add a command to generate a `APP_SECRET` (using the `App\Utils\Random` class)
- Add production assets to the repo
- Add documentation to deploy in production

How to test the feature manually:

1. follow the instructions of the “Deploy in production“ document
2. verify you have a functional Bileto at the end

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
